inc/saiport.h

```diff
@@ -2687,6 +2687,26 @@ typedef enum _sai_port_attr_t
      */
     SAI_PORT_ATTR_PAM4_EYE_VALUES,
 
+    /**
+     * @brief Get port firmware revision
+     * 
+     * Standard attribute to collect port SerDes firmware rev or 
+     * any other vendor specific firmware rev.
+     *
+     * @type sai_s8_list_t
+     * @flags READ_ONLY
+    */
+
+    SAI_PORT_ATTR_FW_REVISION,
```

Comment:
Why is the version a list data type?
Typically if there is a major and minor that can be depicted using uint_32 or as a simple char.

Reply:
Quick scrub of the typial versioning I see something like x.y.z.
Can you give an example and how these attributes should be used i.e. provide a workflow.